packages/build/src/plugins_core/functions/index.js

```diff
@@ -8,26 +8,32 @@ const { isDirectory } = require('path-type')
 
 const { addErrorInfo } = require('../../error/info')
 const {
-  logExperimentalEsbuildParameter,
+  logBundleResults,
   logFunctionsNonExistingDir,
   logFunctionsToBundle,
 } = require('../../log/messages/core_commands')
 
 const { getZipError } = require('./error')
 
-const getZISIParameters = ({ childEnv, logs }) => {
-  // @todo Adjust when experimental support for esbuild is replaced by an
-  // official implementation.
-  const useEsbuild = Boolean(childEnv.NETLIFY_EXPERIMENTAL_ESBUILD)
-  const externalModules = childEnv.NETLIFY_EXPERIMENTAL_EXTERNAL_MODULES
-    ? childEnv.NETLIFY_EXPERIMENTAL_EXTERNAL_MODULES.split(',')
```

Comment:
Should we enable the feature flag for users who currently rely on `NETLIFY_EXPERIMENTAL_EXTERNAL_MODULES`?

Reply:
Yes, I think that's a good idea. Although I know of at least one customer (from https://github.com/netlify/cli/issues/1838) who's not using our build system, and is instead building locally using the CLI. I don't know yet how we'll coordinate the release process to make sure those folks are catered for, but I'll start a discussion for this elsewhere.

Reply:
In the long-term, we should integrate LaunchDarkly with the CLI (https://github.com/netlify/cli/issues/1913).